pkg/agent/controller/trafficcontrol/controller.go

```diff
@@ -151,17 +154,18 @@ func NewTrafficControlController(ofClient openflow.Client,
 	podUpdateSubscriber channel.Subscriber) *Controller {
 	c := &Controller{
 		ofClient:                   ofClient,
+		CrdClient:                  crdClient,
 		ovsBridgeClient:            ovsBridgeClient,
 		ovsCtlClient:               ovsCtlClient,
 		interfaceStore:             interfaceStore,
 		trafficControlInformer:     tcInformer.Informer(),
 		trafficControlLister:       tcInformer.Lister(),
 		trafficControlListerSynced: tcInformer.Informer().HasSynced,
 		podInformer:                podInformer,
-		podLister:                  corelisters.NewPodLister(podInformer.GetIndexer()),
+		PodLister:                  corelisters.NewPodLister(podInformer.GetIndexer()),
 		podListerSynced:            podInformer.HasSynced,
 		namespaceInformer:          namespaceInformer.Informer(),
-		namespaceLister:            namespaceInformer.Lister(),
+		NamespaceLister:            namespaceInformer.Lister(),
```

Comment:
I don't get the changes.

Reply:
These changes were from previous commits, Thanks for pointing out, I will change the case.